bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/tools/ManageEspIdfVersionsHandler.java

```diff
@@ -38,16 +39,30 @@ private void launchEditor()
 			public void run()
 			{
 				IWorkbenchWindow activeww = EclipseHandler.getActiveWorkbenchWindow();
+				if (activeww != null)
+				{
+					IWorkbenchPage page = activeww.getActivePage();
+					if (page != null)
+					{
+						ViewPart viewPart = (ViewPart) page.findView("org.eclipse.ui.internal.introview");
+						if (viewPart != null)
+						{
+							page.hideView(viewPart);
+						}
```

Comment:
Nit: ViewPart casting is not necessary here. We can use IViewPart instead of ViewPart. Also, we don't need if (viewPart != null), because it's a part of the page.hideView(IViewPart) method.